2024/survey.yaml

```diff
@@ -105,15 +105,13 @@ questions:
   - prompt: Which of these best describe your involvement with the Nix ecosystem?
     type: multiple
     choices:
-      - I don't use NixOS
-      - I use NixOS
```

Comment:
These options were added here because we previously had a specific question which [I removed](https://github.com/GuillaumeDesforges/nix-survey/commit/8740f6e77eb0b5c411105c12a315b662fb132148#diff-cfa156da06dd23de4ca97d2568946d71b285dcfcd4b1cec665b227a8c3c19710L256-L260), since we could just merge them all in here.

I still think it is important to gauge NixOS usage specifically and I think this is the only place that was asking about it.

Reply:
To elaborate: the other question (about what is preventing someone from using NixOS) is a free-form question which shouldn't be taken as a signal of NixOS usage specifically. Not everyone will answer a free-form question like that, so the assumption from an answer doesn't hold. (The assumption being: people who answer the question definitely don't use NixOS, people who don't answer use NixOS)

Reply:
I was actually thinking about this one:
https://github.com/GuillaumeDesforges/nix-survey/blob/918848e12f6e530971752eb74295a038c2db430f/2024/survey.yaml#L255

Reply:
I think it has a similar issue with the assumption there. For example, what happens if someone uses NixOS in a machine, but some other distro/OS on a different machine, and they choose to answer based on that other machine? (for what it's worth, this is my case and I'd have answered "with another package manager", even though I consider myself a NixOS user)

This isn't the only issue, it's just an example that points that gauging NixOS usage indirectly through these questions doesn't give us a good signal. Which is why I think we should gauge NixOS usage specifically, because one of the assumptions we want to test is that some users of NixOS have needs that are different from non-NixOS users (and will also help us figure out how large is this share of the audience).

Reply:
That makes sense